about.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hector's Website</title>
+    <link rel="stylesheet" href="styles.css" />
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" />
+</head>
+<header>
+    <nav class="nav-about">
+        <input type="checkbox" id="check">
+        <label for="check" class="checkbtn">
+            <i class="fas fa-bars"></i>
+        </label>
+        <label class="logo"> Hector Pulido</label>
+        <ul>
+            <li>
+                <a class="active" href="index.html">
+                    <h3>HOME</h3>
+                </a>
+            </li>
+            <li>
+                <a href="about.html">
+                    <h3>ABOUT</h3>
+                </a>
+            </li>
+            <li>
+                <a href="experience.html">
+                    <h3>Experience</h3>
+                </a>
+            </li>
+            <li>
+                <a href="contact.html">
+                    <h3>CONTACT</h3>
+                </a>
+            </li>
+        </ul>
+    </nav>
+</header>
+
+<body id="background-about-page">
+    <main class="main-content">
+        <div class="site-container">
```

Comment:
consider using other semantic tags besides div in order to make your page structure more accessible!